src/main/java/seedu/mtracker/console/InputParser.java

```diff
@@ -234,29 +251,6 @@ private void getAndValidateDoneStatus(String[] commandComponents, ArrayList<Inst
         Validate.checkIsNotDone(instruments, instrumentNumber);
     }
 
-    /**
-     * Checks and prepare the find command when a user wants to find instruments.
-     *
-     * @param commandComponents A string array containing the command words the user gave.
-     * @return A command to find instruments.
-     * @throws InvalidEmptySearchStringError If the user does not provide a search string.
-     */
-    public FindCommand getFindInstrumentsCommand(String[] commandComponents)
-            throws InvalidEmptySearchStringError {
-        FindCommand findCommand = new FindCommand();
-        constructSearchString(commandComponents);
-        findCommand.setSearchString(searchString);
-        return findCommand;
-    }
-
-    /**
-     * Filters and returns the command type based on the user input.
-     *
-     * @param commandComponents A string array containing the command words the user gave.
-     * @param instruments The list of instruments currently in the user's watchlist.
-     * @return The command the user wants to execute.
-     * @throws Exception If any of the user's addition or lack of inputs violates the command requirements.
-     */

```

Comment:
I dont think this was deleted by accident? Can help check

Reply:
Oops so sorry. Don't know why it got deleted. Have added back in.